profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -114,74 +116,91 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
-        if (isset($options[$langcode]) && isset($existing_jobs[$langcode])) {
-          $item = $existing_jobs[$langcode];
+        if (isset($options[$lang_code]) && isset($existing_jobs[$lang_code])) {
+          $item = $existing_jobs[$lang_code];
           $uri = $item->uri();
           $wrapper = entity_metadata_wrapper('tmgmt_job_item', $item);
-          $form['languages']['#options'][$langcode][4] = l($wrapper->state->label(),
+          $form['languages']['#options'][$lang_code][TMGMT_WORKBENCH_REQUEST_STATE_INDEX] = l($wrapper->state->label(),
             $uri['path'],
             array('query' => array('destination' => current_path())));
           // Disable the checkbox for this row since there is already a
           // translation in progress that has not yet been finished. This way we
           // make sure that we don't stack multiple active translations for the
           // same item on top of each other.
-          $form['languages'][$langcode] = array(
+          $form['languages'][$lang_code] = array(
             '#type' => 'checkbox',
             '#disabled' => TRUE,
           );
         }
-
         // Disable checkbox selection if current node revision is not
         // in translatable moderation states.
         if (!$translation_possible) {
-          $form['languages'][$langcode] = array(
+          $form['languages'][$lang_code] = array(
             '#type' => 'checkbox',
             '#disabled' => TRUE,
           );
           $form['languages']['#suffix'] = '<div class="description">' .
             t('This content cannot be translated.') . '</div>';
 
-          if (!$always_show_direct_translation_links && ($langcode != $node->translations->original)) {
-            $options[$langcode][5] = '';
+          if ($always_show_direct_translation_links && ($lang_code != $node->translations->original)) {
+            $options[$lang_code][TMGMT_WORKBENCH_REQUEST_OPERATION_INDEX] = '';
           }
         }
       }
 
-      if ($translation_possible) {
+      // Provide 'Request translation' action or hide actions.
+      // NEXTEUROPA-7719 - Simplifying Poetry UI - as a part of this ticket
+      // there is new #markup element which will be displayed every time when
+      // there is a translation request on going.
+      if ($translation_possible && !$existing_jobs) {
         $form['actions']['#type'] = 'actions';
-        $form['actions']['request'] = array(
+        $form['actions']['request'] = [
           '#type' => 'submit',
           '#value' => t('Request translation'),
-          '#submit' => array('tmgmt_workbench_translate_form_submit'),
-          '#validate' => array('tmgmt_workbench_translate_form_validate'),
-        );
+          '#submit' => ['tmgmt_workbench_translate_form_submit'],
+          '#validate' => ['tmgmt_workbench_translate_form_validate'],
+        ];
       }
       else {
         unset($form['actions']);
+        foreach ($options as $lang_code => $option) {
```

Comment:
Same remark as @jfhovinne concerning $option that is never used in the loop. Theuse of "array_keys" could clear the code, IMO.